package.json

```diff
@@ -91,8 +90,8 @@
     "build-size": "rollup -c --silent && gzip-size micromark.min.js && gzip-size --raw micromark.min.js",
     "build": "npm run build-workspace && npm run build-monorepo && npm run build-size",
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
-    "pretest-fuzz": "patch-package --patch-dir script/patches && esbuild micromark --format=cjs --platform=node --bundle > test/fuzz-bundle.cjs && node script/generate-fixtures.js",
-    "test-fuzz": "timeout 30m jsfuzz test/fuzz.cjs test/fixtures",
+    "pretest-fuzz": "esbuild micromark --format=cjs --platform=node --bundle > test/fuzz-bundle.cjs && node script/generate-fixtures.js",
+    "test-fuzz": "jazzer test/fuzz.cjs test/fixtures -- -max_total_time=1800",
```

Comment:
jazzer includes total time feature, so it no longer needs to be wrapped with `timeout`